stdlib/MbedTLS_jll/test/runtests.jl

```diff
@@ -0,0 +1,10 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+using Test, Libdl, MbedTLS_jll
+
+@testset "MbedTLS_jll" begin
+    vstr = zeros(UInt8, 32)
+    ccall((:mbedtls_version_get_string, libmbedcrypto), Cvoid, (Ref{UInt8},), vstr)
+    vn = VersionNumber(unsafe_string(pointer(vstr)))
+    @test vn == v"2.24.0"
```

Comment:
These very strict version checks are problematic with `USE_SYSTEM_MBEDTLS=1` and equivalents, as the system version is unlikely to match exactly. Would it be OK if we replaced this check with a comparison against the minimum required version?